Riot/Modules/TabBar/MasterTabBarController.h

```diff
@@ -178,5 +180,6 @@
 @protocol MasterTabBarControllerDelegate <NSObject>
 
 - (void)masterTabBarControllerDidCompleteAuthentication:(MasterTabBarController *)masterTabBarController;
+- (void)masterTabBarController:(MasterTabBarController*)masterTabBarController wantsToPresentDetailViewController:(UIViewController*)detailViewController;
```

Comment:
The word `present` may be confusing. Maybe we can change this method something like to `- (void)masterTabBarController:(MasterTabBarController*)masterTabBarController wantsToDisplayDetailViewController:(UIViewController*)detailViewController`.